### ISSUE TITLE
Error on import / build, regression since 1.9.5

### ISSUE THREAD

**Issue by msand** [taye/interact.js#803]
### Expected behavior
import interactjs, no exceptions thrown

### Actual behavior
exception is thrown

### System configuration
**interact.js version**: 1.9.5-1.9.7 are broken, works in 1.9.4
**Browser name and version**: any
**Operating System**: any

https://github.com/msand/next-interactjs/blob/629c1da5a669a4fe56ed494a9a76d98a32adc8d3/pages/index.js#L1-L5

next-interactjs/pages/index.js 
```jsx
import interact from "interactjs";

const Home = () => <p>Test interact import</p>;

export default Home;
```

**Comment by msand** [comment]
This seems to work without crashing

```jsx
import '@interactjs/actions/use'
import '@interactjs/auto-start/use'
import '@interactjs/modifiers/use'
import '@interactjs/dev-tools/use'

import interact from '@interactjs/interact'

const Home = () => <p>Test interact import</p>;

export default Home;

```

Adding 
```
import '@interactjs/inertia/use'
```
causes:
```
TypeError: Cannot read property 'prototype' of undefined
    at Object.install (/Users/msand/git/next-interactjs/.next/server/static/development/pages/index.js:7344:23)
    at Scope.usePlugin (/Users/msand/git/next-interactjs/.next/server/static/development/pages/index.js:4858:14)
    at Object.install (/Users/msand/git/next-interactjs/.next/server/static/development/pages/index.js:5196:9)
    at Scope.usePlugin (/Users/msand/git/next-interactjs/.next/server/static/development/pages/index.js:4858:14)
    at Function.use (/Users/msand/git/next-interactjs/.next/server/static/development/pages/index.js:2834:16)
    at Module../node_modules/@interactjs/inertia/use/index.js (/Users/msand/git/next-interactjs/.next/server/static/development/pages/index.js:5595:60)
```

```jsx
import '@interactjs/actions/use'
import '@interactjs/auto-start/use'
import '@interactjs/modifiers/use'
import '@interactjs/dev-tools/use'

import '@interactjs/interact'
import interact from "interactjs";

const Home = () => <p>Test interact import</p>;

export default Home;

```

Causes: 
```
TypeError: Cannot read property 'docEvents' of undefined
    at Object.__install_25 [as install] (/Users/msand/git/next-interactjs/.next/server/static/development/pages/index.js:8852:20)
    at Scope.usePlugin (/Users/msand/git/next-interactjs/.next/server/static/development/pages/index.js:8533:2745)
    at Function.use (/Users/msand/git/next-interactjs/.next/server/static/development/pages/index.js:8583:92)
    at /Users/msand/git/next-interactjs/.next/server/static/development/pages/index.js:9272:3533
    at /Users/msand/git/next-interactjs/.next/server/static/development/pages/index.js:8447:38
    at Object../node_modules/interactjs/dist/interact.min.js (/Users/msand/git/next-interactjs/.next/server/static/development/pages/index.js:8447:60)
    at __webpack_require__ (/Users/msand/git/next-interactjs/.next/server/static/development/pages/index.js:23:31)
    at Module../pages/index.js (/Users/msand/git/next-interactjs/.next/server/static/development/pages/index.js:9302:68)

```

Specifically due to removing the init method from the @interactjs/interactjs/index.ts file in the #800 tree shaking work https://github.com/taye/interact.js/blob/d9db69d6c8f21d6d49856420b42a0986d8d74989/%40interactjs/interactjs/index.ts

**Comment by stahlmanDesign** [comment]
Thanks for looking into this @msand
For now I will use 1.9.4. I am also using it with react-calendar-timeline

**Comment by taye** [comment]
I've tried to reproduce this issue in a project created with `create-react-app`, but I get no errors. What's your build setup?

**Comment by stahlmanDesign** [comment]
> I've tried to reproduce this issue in a project created with `create-react-app`, but I get no errors. What's your build setup?

I also checked it today in CRA and 1.9.9 works fine. But in a Nextjs project it does not build and I and sticking to 1.9.4

**Comment by taye** [comment]
Fixed in v1.9.10

**Comment by stahlmanDesign** [comment]
Thanks, I can confirm the fix works
